style/components/comments.css

```diff
@@ -0,0 +1,55 @@
+.comments {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+}
+
+.comments .textarea {
+  border: 1px solid var(--color-border);
+  border-radius: var(--border-radius);
+  padding: 0.55rem;
+  transition: border-color var(--transition-duration);
+}
+
+.comments .textarea:focus-within {
+  border-color: var(--color-primary);
+}
+
+.comments .textarea textarea {
+  border: none;
+  resize: none;
+  padding: 0 0.4rem;
+}
+
+.comments .textarea .actions {
+  display: flex;
+}
+
+.comments .textarea .actions .submit {
+  margin-right: auto;
+}
+
+.comments .attachments,
+.comments .comment {
```

Comment:
I see you're trying to combine selectors here to reduce code amount. But you have one `display: flex` floating around. Either combine them all, or don't at all.